aldryn_jobs/forms.py

```diff
@@ -146,11 +146,11 @@ class Meta:
             fields.append('content')
 
     def __init__(self, *args, **kwargs):
-        super(JobOfferAdminForm, self).__init__(*args, **kwargs)
+        super(JobOpeningAdminForm, self).__init__(*args, **kwargs)
 
         # small monkey patch to show better label for categories
-        def label_from_instance(obj):
-            return "{0} / {1}".format(obj.category.app_config, obj)
+        def label_from_instance(category_object):
+            return "{0} / {1}".format(category_object.category.app_config, category_object)
```

Comment:
so should i close #80 ? 
also as i've just discovered - flake8 doesn't likes this line length...